src/components/init-modal/style.scss

```diff
@@ -318,12 +318,22 @@
 					border-color: var( --wp-admin-theme-color );
 				}
 
+				&:hover {
+					.block-editor-block-preview__container {
+						margin-top: -25%;
```

Comment:
Non-blocking: Is the intention to show more of the layout as you're hovering over the thumbnail? If so, maybe we should bump this to 50% or even 100%?

Reply:
It is, yes. The reason I used 25% is because the "Support" layout is very short and if I increase the margin then it looks half empty when I'm hovering.